cmd/snap/cmd_debug_state.go

```diff
@@ -80,12 +85,14 @@ func init() {
 		return &cmdDebugState{}
 	}, timeDescs.also(map[string]string{
 		// TRANSLATORS: This should not start with a lowercase letter.
-		"change":    i18n.G("ID of the change to inspect"),
-		"task":      i18n.G("ID of the task to inspect"),
-		"dot":       i18n.G("Dot (graphviz) output"),
-		"no-hold":   i18n.G("Omit tasks in 'Hold' state in the change output"),
-		"changes":   i18n.G("List all changes"),
-		"is-seeded": i18n.G("Output seeding status (true or false)"),
+		"change":      i18n.G("ID of the change to inspect"),
+		"task":        i18n.G("ID of the task to inspect"),
+		"dot":         i18n.G("Dot (graphviz) output"),
+		"no-hold":     i18n.G("Omit tasks in 'Hold' state in the change output"),
+		"changes":     i18n.G("List all changes"),
+		"connections": i18n.G("List all connections"),
+		"connection":  i18n.G("Show details of the matching connections (snap or snap:plug,snap:slot or snap:plug-or-slot"),
```

Comment:
Missing space before comma.

```suggestion
		"connection":  i18n.G("Show details of the matching connections (snap or snap:plug, snap:slot or snap:plug-or-slot"),
```

Reply:
This is intended atm as connection goes as a single argument of `--connection=plug1:slot1,plug2:slot2`; I can rework this if deemed appropriate at the cost of complicating snap debug state args handling.